app/controllers/calculators/adjustable_income_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Calculators
+  class AdjustableIncomeController < CalculatorsController
+    def show
+      @form = AdjustableIncomeForm.new(form_params)
+
+      render partial: 'calculators/adjustable_income/calculator',
+             locals: { form: @form },
+             status: (@form.invalid? ? :bad_request : :ok) if request.xhr?
```

Comment:
Promoting that postfix modifier to a regular conditional would clarify this a little. I think it's hidden away over there.

Also, extracting that ternary expression out to a separate line / assignment makes this easier to scan / parse.